### ISSUE TITLE
Abacus.animation api needs tests for each api method

### ISSUE THREAD

**Issue by rwaldron** [Abacus/Abacus#56]

**Comment by boazsender** [comment]
Rick, in commit 23d100b, I see 5 Abacus.animation property checks, 5 Abacus.animation.layer property checks and 1 test that checks to see that Abacus.animation() fails when passed no opts. 

Can this issue be closed?

**Comment by rwaldron** [comment]
No, I'm looking for new tests to be added that actually test the functionality using as many variants of the api as possible

**Comment by boazsender** [comment]
@mzgoddard, could you introduce new tests to cover more variants of the api?
